plaso/cli/helpers/event_filters.py

```diff
@@ -108,17 +109,22 @@ def ParseOptions(cls, options, configuration_object):
 
     time_slice_event_timestamp = None
     if time_slice_event_time_string:
-      # Note self._preferred_time_zone is None when not set but represents UTC.
-      preferred_time_zone = getattr(
-          configuration_object, '_preferred_time_zone', None) or 'UTC'
-      timezone = pytz.timezone(preferred_time_zone)
-      time_slice_event_timestamp = timelib.Timestamp.FromTimeString(
-          time_slice_event_time_string, timezone=timezone)
-      if time_slice_event_timestamp is None:
+      if ' ' in time_slice_event_time_string:
         raise errors.BadConfigOption(
-            'Unsupported time slice event date and time: {0:s}'.format(
+            'Time slice date and time must be defined in ISO 8601 format.')
+
+      date_time = dfdatetime_time_elements.TimeElements()
+
+      try:
+        date_time.CopyFromStringISO8601(time_slice_event_time_string)
+      except ValueError:
+        raise errors.BadConfigOption(
+            'Unsupported time slice date and time: {0:s}'.format(
```

Comment:
I think it's worth adding an example of what the format is supposed to look like here as well, as this message is directly visible to the user.

Reply:
ack